src/assembly/extension.xml

```diff
@@ -0,0 +1,22 @@
+<assembly xmlns="http://maven.apache.org/ASSEMBLY/2.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/ASSEMBLY/2.0.0 http://maven.apache.org/xsd/assembly-2.0.0.xsd">
+    <id>openrefine-extension-archive</id>
+    <includeBaseDirectory>false</includeBaseDirectory>
```

Comment:
```suggestion
    <baseDirectory>openrefine-extension-archive</baseDirectory>
    <includeBaseDirectory>true</includeBaseDirectory>
```

The above suggestion fixes the issue completely.  That way it actually also coincides with the installation docs as well quite nicely!  In our docs we were telling users themselves to make a top-level folder for the extension and then extract its contents into that.  This is a better way.  With my suggestion here in extension.xml , this automatically makes that top level base directory folder inside the zip and then the `module/` folder shows inside that.  So the user doesn't have to!